bin/invoke_customizations_ou.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+if [ $# -eq 0 ]; then
+  echo "An organizational unit (OU) must be provided as an argument"
+  exit 1
+fi
+read -r -d '' INPUT <<EOF
+  {
+    "include" : [
+      {
+        "type": "ous",
+        "target_value": ["$@"]
+      }
+    ],
+    "exclude" : [
```

Comment:
I don't think we want to exclude prod as we may want to use this to run against prod